util/execdetails.go

```diff
@@ -329,6 +329,10 @@ type ExecDetails struct {
 	BackoffDuration    int64
 	WaitKVRespDuration int64
 	WaitPDRespDuration int64
+	WaitRUDuration     int64
+	// total read ans write ru in 1/1000
+	MilliRRU int64
+	MilliWRU int64
```

Comment:
where does it use in this pr?

Reply:
See: https://github.com/pingcap/tidb/pull/49067